drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/SettingsViewImpl.java

```diff
@@ -97,12 +98,18 @@
     @DataField("stateless")
     protected InputElement stateless = Document.get().createCheckInputElement();
 
-    public SettingsViewImpl() {
-    }
+    @DataField("statelessLabel")
+    protected SpanElement statelessLabel = Document.get().createSpanElement();
 
     @Override
     public void init(Presenter presenter) {
         this.presenter = presenter;
+        nameLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.name());
+        typeLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.type());
+        statelessLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.statelessLabel());
+        dmnFileLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.dmnModelLabel());
+        dmnNamespaceLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.dmnNamespaceLabel());
+        dmnNameLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.dmnNameLabel());
```

Comment:
How about add this check too?
chooseButton.setTitle(ScenarioSimulationEditorConstants.INSTANCE.chooseFile());